src/components/SearchBar/SearchPopup.js

```diff
@@ -45,6 +49,8 @@ function SearchPopup() {
   const classes = useStyles();
   const isActive = useMediaQuery("(max-width:" + breakpoints.values.sm + "px)");
   const [anchorEl, setAnchorEl] = React.useState(null);
+  const [popular, setPopular] = React.useState([]);
+  const [recentSearch, setrecentSearch] = React.useState();
```

Comment:
Instead do this `const [recentSearch, setRecentSearch] = useState();` and import useState at the top.